homeworks/rostyslav.vyshemirskyi_Rooosss/PokemonProject/login.html

```diff
@@ -0,0 +1,59 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+
+        <link rel="stylesheet" href="style.css">
+        <title>PokemonProject</title>
+    </head>
+    <body>
+        <header class="header">
+            <div class="container header__container">
+                <div class="header__inner">
+                    <img src="./image/logo.svg" alt="Logo">
+                    <nav class="navigation">
+                        <ul class="navigation__list">
+                            <li class="navigation__item">
+                                <a class="navigation__link" href="pokemons.html?_ijt=7j4qnubtedq9o7imka7390jf82&_ij_reload=RELOAD_ON_SAVE">Pokemons</a>
+                            </li>
+                            <li class="navigation__item">
+                                <a class="navigation__link" href="index.html">About</a>
+                            </li>
+                            <li class="navigation__item">
+                                <a class="navigation__link" href="#">Login</a>
+                            </li>
+                        </ul>
+                    </nav>
+                </div>
+            </div>
+        </header>
+        <main class="main">
+            <div class="container">
+                <section class="login">
+                    <div class="login__inner">
+                        <h1 class="login__title">Login page</h1>
+                        <form class="login__form" action="success.html?_ijt=jb0rv5r0161gik9qc67rsoff40&_ij_reload=RELOAD_ON_SAVE">
+                            <label class="login__form-label" for="email">Email</label>
+                            <br>
```

Comment:
Now you are using `margin` & `padding` for the gaps between elements. So you don't need these `<br>`s
delete please in whole files